### DISCUSSION TITLE
nCount_RNA

### DISCUSSION THREAD

**Discussion by hyeonjin-cho** [satijalab/seurat#4783] [Q&A]
Hi, I'm working with HCT116 cell line that is processed in 10X platform. 
I'm running basic Seurat QC protocol to filter out the outlying cells that might be doublets or dying. 
However, when I drew the violin plot using:

```
VlnPlot(HCT_T0_DMSO_seurat, features="nCount_RNA", ncol=1)
```

it gives me the plot that I attached, which looks like to me is a bimodal distribution. 
I'm wondering if this is something that sometimes happens with single-cell RNA-seq? 
I have worked with several scRNA-seq data with different cell lines, but I've never seen this kind before (and so haven't my PI and collaborators). 
So from the plot, I would filter out the cells that have less than 3000 features and more than 5000 features. However, even after the filtration, I still see the bimodal distribution on nCount_RNA plot. In this case, should I keep including all the cells (FYI, I'm in the process of checking the quality of the experiment with my collaborator)? 

Also, what do outlying cells in nCount_RNA plot mean?

Thank you in advance!

[HCT_T120_DMSO_VlnPlot.pdf](https://github.com/satijalab/seurat/files/6826689/HCT_T120_DMSO_VlnPlot.pdf)

**Comment by timoast** [comment]
It is unusual to see such a bimodal distribution, but without knowing anything about the experiment I can't really say if this is expected or not or whether those cells should be kept.

Outlier cells for `nCount_RNA` received many more counts than most other cells. This could be due to chance, could be an artefact (eg doublets), or could be due to some other reason

**Comment by ishwarvh** [comment]
> Hi, I'm working with HCT116 cell line that is processed in 10X platform. I'm running basic Seurat QC protocol to filter out the outlying cells that might be doublets or dying. However, when I drew the violin plot using:
> 
> ```
> VlnPlot(HCT_T0_DMSO_seurat, features="nCount_RNA", ncol=1)
> ```
> 
> it gives me the plot that I attached, which looks like to me is a bimodal distribution. I'm wondering if this is something that sometimes happens with single-cell RNA-seq? I have worked with several scRNA-seq data with different cell lines, but I've never seen this kind before (and so haven't my PI and collaborators). So from the plot, I would filter out the cells that have less than 3000 features and more than 5000 features. However, even after the filtration, I still see the bimodal distribution on nCount_RNA plot. In this case, should I keep including all the cells (FYI, I'm in the process of checking the quality of the experiment with my collaborator)?
> 
> Also, what do outlying cells in nCount_RNA plot mean?
> 
> Thank you in advance!
> 
> [HCT_T120_DMSO_VlnPlot.pdf](https://github.com/satijalab/seurat/files/6826689/HCT_T120_DMSO_VlnPlot.pdf)

Hey, 

Were you able to figure out this issue? I am seeing a bimodal distribution of nCount and nFeatures in my dataset as well.
The doublet prediction software aren't identifying these as doublets.

One of the possibilities could be that there were some nuclei present in the single-cell suspension that was used to make the GEM's and therefore single nuclei and single cells were sequenced in this experiment

That's just an idea, but I am wondering if anyone else has a better explanation hopefully with some empirical evidence!
